New eBPF option (#14691)
netdata/netdata: collectors/ebpf.plugin/README.md

`ebpf.d.conf`, and then disable the plugin for a specific `thread` by following the steps in the
[Configuration](#configuring-ebpfplugin) section.

#### Maps per Core

> When netdata is running on kernels newer than `4.6` users are allowed to modify how `ebpf.plugin` creates maps (hash or 

Reviewer: ```suggestion
When netdata is running on kernels newer than `4.6` users are allowed to modify how the `ebpf.plugin` creates maps (hash or 
```
```suggestion
When netdata is running on kernels newer than `4.6` users are allowed to modify how the `ebpf.plugin` creates maps (hash or 
```

will only enable these threads integrated with other collectors when the kernel is compiled with
`CONFIG_DEBUG_INFO_BTF`, unless you enable them manually.

#### Collection period

> Plugin uses option `update every` to define number of seconds used for eBPF to send data for Netdata. The default value

Reviewer: ```suggestion
The plugin uses the option `update every` to define number of seconds used for eBPF to send data for Netdata. The default value
```

> The plugin uses the option `update every` to define number of seconds used for eBPF to send data for Netdata. The default value

Reviewer: ```suggestion
The plugin uses the option `update every` to define the number of seconds used for eBPF to send data for Netdata. The default value
```
```suggestion
The plugin uses the option `update every` to define number of seconds used for eBPF to send data for Netdata. The default value
```

is 5 seconds.

#### PID table size

> The option `pid table size` defines the maximum number of PIDs stored inside the application hash table. Default value

Reviewer: ```suggestion
The option `pid table size` defines the maximum number of PIDs stored inside the application hash table. The default value
```

eBPF monitoring is complex and produces a large volume of metrics. We've discovered scenarios where the eBPF plugin
significantly increases kernel memory usage by several hundred MB.

> When the integration with apps or cgroup is enabled, eBPF collector allocates memory for each process running. If your

Reviewer: ```suggestion
When the integration with apps or cgroup is enabled, the eBPF collector allocates memory for each process running. If your
```

node is experiencing high memory usage and there is no obvious culprit to be found in the `apps.mem` chart, consider:

- Modify [maps per core](#maps-per-core) to use only one map.
- Disable [integration with apps](#integration-with-appsplugin).
- Disable [integration with cgroup](#integration-with-cgroupsplugin).

> If with these changes you still consider plugin having a high memory usage, and there is no obvious culprit to be found 

Reviewer: ```suggestion
If with these changes you still suspect eBPF using too much memory, and there is no obvious culprit to be found 
```
